src/adapters/scene.js

```diff
@@ -81,6 +89,9 @@ Scene.prototype.initMapBox = async function (locationHash) {
     locale,
     ...(await this.getMapInitOptions(locationHash)),
   });
+  // @MAPBOX: This method isn't implemented by the Mapbox-GL mock
+  this.mb.setPadding = this.mb.setPadding || (() => {});
```

Comment:
we might check for the NODE_ENV variable to be "test" ?

Reply:
Do you mean wrapping that in `if (process.env.NODE_ENV === 'test') { … }`?

Unfortunately it doesn't work with our current setup, as the test version is built with the `production` mode, so Webpack just removes the block from the build and the fix doesn't get applied.
I guess it deserves a talk about our test practices, maybe we could change that, but let's do it separately from this PR.